UnitTests/BraintreeCoreTests/Configuration/ConfigurationLoader_Tests.swift

```diff
@@ -132,11 +123,16 @@ class ConfigurationLoader_Tests: XCTestCase {
         }
     }
     
-    func testGetConfig_whenCalledInQuickSequence_onlySendsOneNetworkRequest() {
-        sut.getConfig() { _, _ in }
-        sut.getConfig() { _, _ in }
-        sut.getConfig() { _, _ in }
-        sut.getConfig() { _, _ in }
+    func testGetConfig_whenCalledInQuickSequence_onlySendsOneNetworkRequest() async {
+        do {
+            async let functionOne = sut.getConfig()
+            async let two = sut.getConfig()
+            async let three = sut.getConfig()
+            async let four = sut.getConfig()
+            let _ = try await (functionOne, two, three, four)
```

Comment:
@KunJeongPark are you able to walk me through how this would be triggered by a merchant? The `getConfig()` method is internal to the SDK so I am trying to grock how we would end up with this situation.

Reply:
Sure, they wouldn't call this on its own but we are testing this function but itself on this test.
In our SDK we are calling getConfig from Task annotated @MainActor so it ensures that getConfig calls are serialized. But we have a test here that tests that there are no duplicate calls to this function.

So making the test this way instead of awaiting each function call that we had in earlier commit, is really testing if concurrent calls or quick successive calls would not be problematic and makes just one network call.

Reply:
Gotcha - so really just an update to protect ourself from accidentally doing something weird in the future and causing issues. Thanks for the explanation.